documentation/develop.md

```diff
@@ -0,0 +1,86 @@
+# Development
+
+Please as a first start, tell the community about your intent of developing a feature or fix a bug. Search for the associated issue if it exists or open one with steps to reproduce (for bugs) or a [user story](https://en.wikipedia.org/wiki/User_story#Principle) (for features).
+
+## Setup
+
+### Pre-requisites
+
+To setup your environment, you would need to install the following dependencies:
+ - git
+ - [nvm](https://github.com/nvm-sh/nvm/blob/master/README.md)
+ - Firefox and/or Chromium to run the end-to-end (nbrowsers) tests
```

Comment:
I think so, I'll test that.

Reply:
And maybe just say "browser tests" instead of "(nbrowsers) tests".

Reply:
Ok, end-to-end is fine. I was hoping to replace/eliminate `nbrowsers` since it was a bit jargony (user won't know anything about the idiosyncratically named `nbrowser` tests yet) and also the `s` is maybe a bit confusing (the test target it is referring to is `nbrowser`). How about just dropping `(nbrowsers)`?

Reply:
Ah, I see it is already fixed :+1:

Reply:
> Ok, end-to-end is fine

If you're uncomfortable with naming them end-to-end, don't hesitate to tell me. I am not very confident on how to name them ^^